blogposts/2021/better-onboarding-advice-from-3-engineering-leaders.md

```diff
@@ -0,0 +1,127 @@
+
+## Offer emotional support
+
+Even senior engineers can suffer from impostor syndrome. Literally everyone feels intimidated when diving into a new large codebase. 
+Finding the right balance between past experience and new territory is crucial, and organizations that value this should go out of their way to create a welcoming culture and environment.
+
+For new hires, having a marker for "am I living up to expectations" in the first couple of weeks can ease their worries. 
+It's often better to err on the side of having an easier onramp, due to all the pitfalls of onboarding. 
+This also allows an EM to establish a baseline for how a new hire works and what they're capable of. 
+If you overshoot, you'll be unsure if it's a performance issue or if the person just happened to fail due to tough circumstances.
+
+Remember, you hired someone because you thought they were excellent. So if they are struggling, you should figure out what might be throwing them off, or revisit your hiring processes if this is happening often. 
+It's common for fast-growing startups to experience some churn and attrition, but if this is happening a lot, it's important for someone in the org to be asking these questions and sharing as much learning as they can while still respecting people's individual privacy about employment decisions.
+
+## Embrace continuous onboarding
+
+> "I think most people look at onboarding in the same way we look at education: you go to school, you graduate, and you're done with learning. 
+> But the reality is you should never be done with learning. And in the same way, when it comes to onboarding onto codebases and infrastructure and that kind of thing, you should never be done with learning there because those systems aren't in a consistent state; they're evolving over time as the business evolves." – Ryan
+
+This is especially true when a developer switches teams or starts a new project that involves a previously unfamiliar part of the codebase. 
+There are some things they already know and don't need to repeat—for example, they likely already have the dev environment set up and have the big picture of the overall company—but there is still a lot of context to acquire. 
+Therefore, investing in good onboarding can be an investment in the overall acceleration of the development process, because in some sense, devs are continuously onboarding to new domains of knowledge. 
+It is the job of a developer to learn new domains of knowledge and construct systems that link together these different domains. 
+Onboarding is part of the job description.
+
+## Onboarding now and then
+
+It can sometimes be tempting to yearn for the way things were, before the advent of distributed systems, of big complex web applications, of software permeating every company, when codebases were smaller, less interconnected. 
```

Comment:
```suggestion
It can sometimes be tempting to yearn for the way things were when codebases were smaller and less interconnected—before the advent of distributed systems, of big complex web applications, of software permeating every company. 
```